ControlFlowArraysAndStrings/ControlFlowArraysAndStrings/Program.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System;
+
+namespace ControlFlowArraysAndStrings
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string[] fruits = { "banana",
+                                "pineapple",
+                                "apple",
+                                "orange",
+                                "blueberry",
+                                "raspberry",
+                                "apricot" };
+
+            // Original Array.
+            Console.WriteLine("Original Array: ");
+            Print(fruits);
+
+            // Sort method.
+            Console.WriteLine($"{Environment.NewLine}Testing the Sort method:");
+            string[] sortedFruits = Sort(fruits);
+            Print(sortedFruits);
+
+            // AddElementToStart method.
+            Console.WriteLine($"{Environment.NewLine}Testing the AddElementToStart method:");
+            string[] addWatermelon = AddElementToStart(fruits, "watermelon");
+            Print(addWatermelon);
+
+            // AddElementToEnd method.
+            Console.WriteLine($"{Environment.NewLine}Testing the AddElementToEnd method:");
+            string[] addKiwi = AddElementToEnd(fruits, "kiwi");
+            Print(addKiwi);
+
+            // AddElementAnywhere method.
+            Console.WriteLine($"{Environment.NewLine}Testing the AddElementAnywhere method:");
+            string[] addLime = AddElementAnywhere(fruits, 3, "lime");
+            Print(addLime);
+
+            string[] addLime2 = AddElementAnywhere(fruits, 7, "lime");
+            Print(addLime2);
+
+            string[] addLime3 = AddElementAnywhere(fruits, 0, "lime");
+            Print(addLime3);
+
+            // RemoveElementFromStart method.
+            Console.WriteLine($"{Environment.NewLine}Testing the RemoveElementFromStart method:");
+            string[] removeBanana = RemoveElementFromStart(fruits);
+            Print(removeBanana);
+
+            // RemoveElementFromEnd method.
+            Console.WriteLine($"{Environment.NewLine}Testing the RemoveElementFromEnd method:");
+            string[] removeApricot = RemoveElementFromEnd(fruits);
+            Print(removeApricot);
+
+            // RemoveElementFromAnywhere method.
+            Console.WriteLine($"{Environment.NewLine}Testing the RemoveElementFromAnywhere method:");
+            string[] removePineapple = RemoveElementFromAnywhere(fruits, 3);
+            Print(removePineapple);
+
+            string[] removeStart = RemoveElementFromAnywhere(fruits, 0);
+            Print(removeStart);
+
+            string[] removeEnd = RemoveElementFromAnywhere(fruits, 6);
+            Print(removeEnd);
+
+            // Login method.
+            Console.WriteLine($"{Environment.NewLine}Testing the Login method:");
+            Login();
```

Comment:
Instead of those code blocks with comments, you could have the same done in methods and then wouldn't need comments.

instead of:

```cs
            // Sort method.
            Console.WriteLine($"{Environment.NewLine}Testing the Sort method:");
            string[] sortedFruits = Sort(fruits);
            Print(sortedFruits);
```

```cs
            void DemoSortMethod()
            {
                Console.WriteLine($"{Environment.NewLine}Testing the Sort method:");
                string[] sortedFruits = Sort(fruits);
                Print(sortedFruits);
            }
```